test/replicate.js

```diff
@@ -1325,51 +1333,6 @@ test('idle replication sessions auto gc', async function (t) {
   t.ok(closed, 'replication session gced')
 })
 
-test('idle replication sessions auto gc with timing', async function (t) {
```

Comment:
Sanity check: has this test become irrelevant now? Seems like it could be relevant for detecting issues with this PR if it's possible to adapt it

Reply:
ya just too many private things being inspected here so killed it, you'll need to hook into peer._closeSession for this now

Reply:
or mb just on('peer-remove')